WORKSPACE

```diff
@@ -4,6 +4,9 @@ load("//bazel:repositories.bzl", "envoy_dependencies")
 load("//bazel:cc_configure.bzl", "cc_configure")
 
 envoy_dependencies()
+load("@rules_foreign_cc//:workspace_definitions.bzl", "rules_foreign_cc_dependencies")
```

Comment:
I would like to figure out how to roll this into `envoy_dependencies()`, but it depends on some of the setup there. `load` can't be used inside a macro, only at file level or in `WORKSPACE`. Might be possibly with a multi-phase `envoy_dependencies()` setup.

Reply:
afaik, unfortunately you can only call the initialization function directly in WORKSPACE file. Should be resolved when recursive workspace paring be implemented.